OpenRA.Mods.Common/AI/BaseBuilder.cs

```diff
@@ -37,14 +43,59 @@ public BaseBuilder(HackyAI ai, string category, Player p, PowerManager pm, Playe
 			playerPower = pm;
 			playerResources = pr;
 			this.category = category;
+			cachedBases = 0;
```

Comment:
0 / false are the default values for the data types anyway, so these are redundant.